pkg/scheduler/partition.go

```diff
@@ -224,7 +224,7 @@ func (pc *PartitionContext) updateQueues(config []configs.QueueConfig, parent *o
 		queue := pc.getQueueInternal(pathName)
 		var err error
 		if queue == nil {
-			queue, err = objects.NewConfiguredQueue(queueConfig, parent)
+			queue, err = objects.NewConfiguredShadowQueue(queueConfig, parent)
```

Comment:
Is this path intended to update the partition based on a configuration reload? If so, could you explain why it is referred to as a "shadow queue"?

Reply:
@chia7712 thanks for reviewing! Yes, it should not be referred to as a "shadow queue", I completely misunderstood the code here. I'm now working on a new version based on the guidance.